main.go

```diff
@@ -82,6 +83,12 @@ func main() {
 		os.Exit(0)
 	}
 
+	watchNamespace, err := utils.GetWatchNamespace()
+	if err != nil {
+		klog.Error(err, "unable to get WatchNamespace, "+
+			"the manager will watch and manage resources in all namespaces")
+	}
```

Comment:
Woudln't it be ok to watch for all namespaces by default?

Reply:
if https://github.com/ArangoGutierrez/node-feature-discovery-operator/blob/devel/v1spec_ns/pkg/utils/utils.go#L28 is not defined, if will default to all, so is ok.